src/bigint.h

```diff
@@ -38,6 +38,12 @@ class Bigint
     Bigint operator*(long long const &);
     Bigint &operator*=(int const &);
 
+    //Division
+    bool is_even();
+    private:
+    std::vector<Bigint> divide(Bigint q);   // returns quotient(index[0]) and remainder(index[1]).
+    public:
+    std::vector<Bigint> operator/(Bigint q); // interface for divide() function.
```

Comment:
I think there can be both <code>operator/</code> and <code>operator%</code> wrapper <code>divide</code> and split the results, return what is needed, just like killing two birds with one stone. By the way, I think you algorithm is so cool :+1: @MahmoudRizk